ci-operator/config/openshift/oauth-proxy/openshift-oauth-proxy-release-3.11.yaml

```diff
@@ -0,0 +1,27 @@
+base_images:
+  base:
+    name: origin-v3.11
+    namespace: openshift
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+images:
+- from: base
+  to: oauth-proxy
+promotion:
+  name: "3.11"
```

Comment:
I think this should be `origin-v3.11`, thats where e.g. openshift/origin promotes: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/origin/openshift-origin-release-3.11.yaml#L64